test/fixtures.ts

```diff
@@ -221,9 +221,14 @@ class CLIMock {
     this.process.stdout.on('data', line => {
       this.data += removeAnsiColors(line.toString());
       if (this.waitForCallback && this.data.includes(this.waitForText))
-        this.waitForCallback()
+        this.waitForCallback();
     })
-    this.exited = new Promise(r => this.process.on('exit', r))
+    this.exited = new Promise(r => this.process.on('exit', () => {
+      if (this.waitForCallback) {
+        this.waitForCallback();
+      }
+      return r;
```

Comment:
Resolve is a callback, so we should call it.
```suggestion
      return r();
```